src/dlgProfilePreferences.cpp

```diff
@@ -2093,6 +2119,78 @@ void dlgProfilePreferences::slot_logFileNameFormatChange(const int index)
     label_logFileNameExtension->setVisible(isShown);
 }
 
+void dlgProfilePreferences::slot_configDirOptionChange(const int index)
+{
+    Q_UNUSED(index);
```

Comment:
It isn't unused though, is it?! :confused:

Reply:
Fixed